src/components/Input/Input.vue

```diff
@@ -145,7 +132,20 @@ export default defineComponent({
     }
   },
   computed: {
+    outerClassList(): any {
+      const baseList = [this.$attrs.class]
+      return this.disabled
+        ? ['disabled', ...baseList]
+        : this.invalid
+        ? ['invalid', ...baseList]
+        : this.active
+        ? ['active', ...baseList]
+        : this.hovered
+        ? ['hovered', ...baseList]
+        : baseList
+    },
```

Comment:
```suggestion
    outerClassList(): any {
      return [
        ...this.$attrs.class,
        {
          disabled: this.disabled,
          invalid: this.invalid,
          active: this.active,
          hovered: this.hovered
        }
      ]
    },
```

Reply:
Realizing the logic here might not be quite right. The original will bind `disabled` but not `invalid` if you pass in both props. Which seems odd to me 🤔